src/api/index.js

```diff
@@ -25,3 +25,5 @@ export const createPost = post => api.post(`/posts`, post).then(extractData);
 
 export const updatePost = post =>
   api.put(`/posts/${post.id}`, post).then(extractData);
+
+export const fetchTodos = () => api.get(`/tod`).then(extractData);
```

Comment:
- "tod"?
- backticks not needed, please use regular quotes

Reply:
Tod was required to show error :-)